src/DiskCache.ts

```diff
@@ -1,32 +1,41 @@
 import fs from 'fs';
-import CacheInterface from './interfaces/CacheInterface.js';
-import IOBase from './IOBase.js';
 import path from 'path';
-import ConfigInterface from 'interfaces/ConfigInterface.js';
+import ConfigInterface from 'interfaces/ConfigInterface';
+import CacheInterface from './interfaces/CacheInterface';
+import IOBase from './IOBase';
+import CacheNotFoundError from './errors/CacheNotFoundError';
+import CacheFileInterface from './interfaces/CacheFileInterface';
 
 class DiskCache extends IOBase {
   constructor(baseDir: string = './.cache') {
```

Comment:
Standaard uitgaan van productie omgeving
```suggestion
  constructor(baseDir: string = '/cache') {
```